PostHog/Utils/UIApplication+.swift

```diff
@@ -26,6 +26,7 @@
                 } else {
                     // check scene.windows.isKeyWindow
                     for window in scene.windows {
+                        // swiftlint:disable:next for_where
```

Comment:
We use `where` in some other places already so let's just use it instead of silencing the linter.